src/pages/workspace/bills/WorkspaceBillsFirstSection.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
-import {View, TouchableOpacity} from 'react-native';
+// eslint-disable-next-line no-restricted-imports
+import {View, TouchableOpacity, Text as RNText} from 'react-native';
```

Comment:
`Text as RNText` is really necessary?

Reply:
Text causes some issues on ipod 7th gen when device font size is increased.

Reply:
`Text as RNText` but the component is still the same we are just using an alias

Reply:
I speaking about our custom `Text` component there https://github.com/Expensify/App/blob/724ccb46ca4b2f48f6052592cbacc7ffe9528c5c/src/pages/workspace/bills/WorkspaceBillsFirstSection.js#L6
we need to replace it with a react native `Text` component because it causes some issues on ipod 7th gen when device font size is increased.